### PR TITLE
EditorConfig file type  #145

### PR DESCRIPTION
**Pull request by HakanVardarr**
I added EditorConfigValidator and 2 test to the project. Does not log errors very well, I tried to do it the way the library supports

### CODE REVIEW

**Review by ccoVeille — APPROVED** [review]

**Comment by ccoVeille** [comment]
Fixes #145

For records, mentioning issue in title does nothing

**Comment by ccoVeille** [comment]
Thanks @HakanVardarr

**Comment by HakanVardarr** [comment]
> Fixes #145
> 
> For records, mentioning issue in title does nothing

Oh didn't know that thanks

File: pkg/filetype/file_type.go [hunk 1]
```diff
@@ -103,6 +103,14 @@ var EnvFileType = FileType{
 	validator.EnvValidator{},
 }
 
+// Instance of the FileType object to
+// represent a EDITORCONFIG file
```
Comment (kehoecj) [inline pkg/filetype/file_type.go:107]:
```suggestion
// represent an EDITORCONFIG file
```

**Review by kehoecj — APPROVED** [review]
LGTM Thanks for another PR @HakanVardarr !

**Comment by kehoecj** [comment]
>Does not log errors very well, I tried to do it the way the library supports

It uses the ini library. We have an open issue to that repo to support line and characters in the error message.

**Pull request merged**